api/base/filters.py

```diff
@@ -292,7 +292,7 @@ def parse_query_params(self, query_params):
                         query.get(key).update({
                             field_name: self._parse_date_param(field, source_field_name, op, value),
                         })
-                    elif not isinstance(value, int) and source_field_name in ['_id', 'guid._id', 'journal_id', 'moderation_state']:
+                    elif not isinstance(value, int) and ',' in value:
```

Comment:
As I mentioned on Slack, let's just add the field we want to explicitly filter on here rather than opening it up to all fields. Too open may cause people to be able to easily construct complex queries that would cause us database problems without having a clear use case for us.